helm/index.yaml

```diff
@@ -1,6 +1,3 @@
-# BEGIN COPYRIGHT NOTICE
-# (c) Copyright 2023 Micro Focus or one of its affiliates.
-# END COPYRIGHT NOTICE
```

Comment:
Please put the copyright notice back in (this is unfortunately a manual step after running `make index.yaml`)

Reply:
[4c70982](https://github.com/opentext-idol/idol-containers-toolkit/pull/19/commits/4c70982140b91848aea12eacfc16280317b286b5)